packages/uiweb/src/lib/components/chat/ChatPreviewSearchList/ChatPreviewSearchList.tsx

```diff
@@ -230,19 +229,21 @@ export const ChatPreviewSearchList: React.FC<IChatPreviewSearchListProps> = (opt
           } else {
             const userProfile = await user?.info({ overrideAccount: chatInfo.recipient });
             console.debug('UIWeb::components::ChatPreviewSearchList::loadMoreChats::userProfile', userProfile);
-
             searchedChat = {
               ...searchedChat,
               chatId: derivedChatId,
-              chatParticipant: derivedChatId,
+              chatParticipant: derivedChatId
+                ? formattedChatId.includes('.')
```

Comment:
Let's put  formattedChatId.includes('.') this in a utility function. Something like

const getChatId=(id:string)=> id.includes(".")? id : null

Reply:
okay

Reply:
@rohitmalhotra1420  added the change